src/Imdb/Title.php

```diff
@@ -477,29 +477,46 @@ public function comment_split() {
  #-------------------------------------------------------[ Recommendations ]---
   /**
    * Get recommended movies (People who liked this...also liked)
-   * @return array recommendations (array[imdbid,title,year])
+   * @return array recommendations (array[title,imdbid,year,endyear,type])
    * @see IMDB page / (TitlePage)
    */
-  public function movie_recommendations() {
-    if (empty($this->movierecommendations)) {
-      $doc = new \DOMDocument();
-      @$doc->loadHTML($this->getPage("Title"));
-      $xp = new \DOMXPath($doc);
-      $cells = $xp->query("//div[@id=\"title_recs\"]/div[@class=\"rec_overviews\"]/div[@class=\"rec_overview\"]/div[@class=\"rec_details\"]");
-      foreach ($cells as $cell) {
-        preg_match('!tt(\d+)!',$cell->getElementsByTagName('a')->item(0)->getAttribute('href'),$ref);
-        $movie['title'] = trim($cell->getElementsByTagName('a')->item(0)->nodeValue);
-        $movie['imdbid'] = $ref[1];
-        preg_match('!(\d+)!',$cell->getElementsByTagName('span')->item(0)->nodeValue,$ref);
-        if (!isset($ref[1])) {
```

Comment:
This was added to account for "Sinister I (2012) " on https://www.imdb.com/title/tt0450385/ It'd been removed from the test pack because I think that recommendation had gone ... but it's back and i've added the test back in

Reply:
Aah now i know why this was in this method in the first place.
I good not figured out what it is used for but now i do!

i'll put it back in asap

Reply:
reworked, added back the check for the right span to account for sinister
"Sinister I (2012) " on https://www.imdb.com/title/tt0450385/

Reply:
This adjustment is only necessary if you use dom/xpath
If you use preg_match you can match the span with class="nobr"

But we discussed earlier that it would be better to less use preg_match and more dom/xpath